kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/BrokersResource.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.resources.v3;
+
+import io.confluent.kafkarest.Versions;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response.Status;
+
+@Path("/v3/clusters/{clusterId}/brokers")
+public final class BrokersResource {
+
+  @GET
+  @Produces(Versions.JSON_API)
+  public void listBrokers(@PathParam("clusterId") String clusterId) {
```

Comment:
I see in v2 we were using @PerformanceMetric, why don't we use it now ?

Reply:
We can add performance metrics to all endpoints later.